logstash-core/src/main/java/org/logstash/ext/JrubyTimestampExtLibrary.java

```diff
@@ -254,5 +248,9 @@ public IRubyObject ruby_year(ThreadContext context)
         {
             return RubyFixnum.newFixnum(context.runtime, this.timestamp.getTime().getYear());
         }
+
+        private static RubyTimestamp fromRString(final Ruby runtime, final RubyString string) {
+            return RubyTimestamp.newRubyTimestamp(runtime, new Timestamp(string.toString()));
```

Comment:
is `string`  guaranteed to always be a non-null ?

I _think_ old implementation would return a `Timestamp` of _now_, where this will result in `NPE`.